azext_iot/digitaltwins/_help.py

```diff
@@ -140,6 +140,112 @@ def load_digitaltwins_help():
             az dt reset -n {instance_name}
     """
 
+    helps["dt data-history"] = """
+        type: group
+        short-summary: Manage and configure data history connections.
+    """
+
+    helps["dt data-history create"] = """
+        type: group
+        short-summary: Creates a data history connection between a Digital Twins instance and another resource.
+    """
+
+    helps["dt data-history create adx"] = """
+        type: command
+        short-summary: Creates a data-history connection between a Digital Twins instance and an Azure Database Explorer.
+            Requires pre-created Azure Database Explorer and Eventhub resources.
+        examples:
+        - name: Adds a data history connection to a target instance with the $Default event hub consumer group.
+          text: >
+            az dt data-history create adx -n {instance_name}
+            --cn {time_series_database_connection_name}
+            --adx-cluster-name {adx_cluster_name}
+            --adx-database-name {adx_database_name}
+            --eventhub {event_hub}
+            --eventhub-namespace {event_hub_namespace}
+        - name: Adds a data history connection to a target instance with a custom Azure Database Explorer table name
```

Comment:
Another Database Explorer verbiage to fix. I won't point out further ones if they exist.